aws/service.go

```diff
@@ -119,6 +120,27 @@ func APIGatewayV2Service(ctx context.Context, d *plugin.QueryData, region string
 	return svc, nil
 }
 
+// ApplicationAutoscalingService returns the service connection for AWS ACM service
+func ApplicationAutoscalingService(ctx context.Context, d *plugin.QueryData, region string) (*applicationautoscaling.ApplicationAutoScaling, error) {
```

Comment:
@johnsmyth Since this requires a new service, and Terraform has its own resource type to get application auto-scaling targets (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/appautoscaling_target), it seems like we should add the aws_appautoscaling_target table and join it against a DDB table's resource ID.

Thoughts?

Reply:
Seems reasonable, especially considering that there are also other tf resources related to app auto-scaling -- aws_appautoscaling_policy, aws_appautoscaling_scheduled_action are probably also separate  tables